editor/editor_about.cpp

```diff
@@ -105,11 +122,46 @@ ScrollContainer *EditorAbout::_populate_list(const String &p_name, const List<St
 			il->set_same_column_width(true);
 			il->set_auto_height(true);
 			il->set_mouse_filter(Control::MOUSE_FILTER_IGNORE);
+			il->set_focus_mode(Control::FOCUS_NONE);
 			il->add_theme_constant_override("h_separation", 16 * EDSCALE);
-			while (*names_ptr) {
-				il->add_item(String::utf8(*names_ptr++), nullptr, false);
+			if (p_allow_website) {
+				il->set_focus_mode(Control::FOCUS_CLICK);
```

Comment:
What is this for? It makes the list impossible to focus with keyboard when you navigate out of it.

Reply:
Tested with FOCUS_ALL again, behavior is even more buggy for some reason, so I personally decided to keep it just for the mouse.